src/sections/newcontactuspage/haveAChat/HaveAChat.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import styles from "./HaveAChat.css";
+import { Button } from "@nextui-org/button";
+import Image from "next/image";
+import { ICONS } from "@/utils/icons";
+
+
+const HaveAChat = () => {
+  return (
+    <div className="Backgroundforchat">
+      <div >
+        <p className="firstparap">Let's Have a Chat!</p>
```

Comment:
Escape `'` in *"Let's Have a Chat!"* with `&apos;`. I.E.: `Let&apos;s Have a Chat!`.